src/providers/twitch/PubsubClient.cpp

```diff
@@ -1428,23 +1431,45 @@ void PubSub::handleMessageResponse(const rapidjson::Value &outerData)
                 rapidjson::Value senderData;
                 if (!rj::getSafeObject(messageData, "sender", senderData))
                 {
-                    qDebug() << "Failed to get sender";
+                    qCDebug(chatterinoPubsub) << "Failed to get sender";
                     return;
                 }
-                QString sender_id;
-                if (!rj::getSafe(senderData, "user_id", sender_id))
+                QString senderId;
+                if (!rj::getSafe(senderData, "user_id", senderId))
                 {
-                    qDebug() << "Failed to get sender user id";
+                    qCDebug(chatterinoPubsub) << "Failed to get sender user id";
                     return;
                 }
-                QString sender_login;
-                if (!rj::getSafe(senderData, "login", sender_login))
+                QString senderLogin;
+                if (!rj::getSafe(senderData, "login", senderLogin))
                 {
-                    qDebug() << "Failed to get sender login";
+                    qCDebug(chatterinoPubsub) << "Failed to get sender login";
                     return;
                 }
-                action.target = ActionUser{sender_id, sender_login};
-                qDebug() << action.msgID;
+                QString senderDisplayName;
+                if (!rj::getSafe(senderData, "display_name", senderDisplayName))
+                {
+                    qCDebug(chatterinoPubsub)
+                        << "Failed to get sender display name";
+                    return;
```

Comment:
Could we just handle this gracefully by setting displayName to login?
EDIT: I see we do comparisons with login below, so we would need to take that into consideration too